portfolio/src/main/webapp/nav.js

```diff
@@ -0,0 +1,9 @@
+const content = '\
```

Comment:
Nice job with the nav bar. This uses purely javascript, but if you'd like to use jQuery it's possible to do this as well: https://medium.com/@AmyScript/how-to-reduce-reuse-and-recycle-your-code-389e6742e4ac

My only reservation with this is that now you have some html that is in a js file instead of an html file. I'm not too sure if this is considered a style issue, might be worth asking a virtual support engineer if there's a way for you to do so if you have time! If you do, would love to hear what they tell you about this.

Reply:
Will do!